security/vm-sudo.md

```diff
@@ -105,7 +105,8 @@ this for extra security.**
 1. Adding Dom0 "VMAuth" service:
 
         [root@dom0 /]# echo -n "/usr/bin/echo 1" >/etc/qubes-rpc/qubes.VMAuth
-        [root@dom0 /]# echo -n "\$anyvm dom0 ask" >/etc/qubes-rpc/policy/qubes.VMAuth
+        [root@dom0 /]# echo -n "\$anyvm dom0 ask,default_target=dom0" \
+        >/etc/qubes-rpc/policy/qubes.VMAuth
```

Comment:
I know it was here already, but why `echo -n`, in both cases? Looks unnecessary.

Reply:
Don't know why. I'll remove it.